### PR TITLE
adding collector for etcd

### PR DESCRIPTION
**Pull request by rtoma**

### CODE REVIEW

**Comment by rtoma** [comment]
how do I add those? any instructions available?

**Comment by shortdudey123** [comment]
This should do it (if not, add `touch doc/collector/EtcdCollector.md`)

```
./build_doc.py -C etcdstat
./test.py -c etcdstat
```

Also might want to rename the class to include `stat` or rename the directory and file to `etcd`

**Comment by rtoma** [comment]
Thanks, I will fix the docs.

I've named the directory etcdstat, to avoid conflicts with the official etcd python module.
I believe its a trick used for many collectors (e.g. https://github.com/python-diamond/Diamond/blob/master/src/collectors/kafkastat/kafkastat.py)?

**Comment by shortdudey123** [comment]
cool

File: docs/collectors/EtcdCollector.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+metrics_blacklist | None | Regex to match metrics to block. Mutually exclusive with metrics_whitelist | NoneType
+metrics_whitelist | None | Regex to match metrics to transmit. Mutually exclusive with metrics_blacklist | NoneType
+port | 2379 | Port (default is 2379) | int
+timeout | 5 | Timeout per HTTP(s) call | int
+use_tls | False | Use TLS/SSL or just unsecure (default is unsecure) | bool
+
+#### Example Output
+
+```
+__EXAMPLESHERE__
```
Comment (shortdudey123) [inline docs/collectors/EtcdCollector.md]: can you run `test.py -c etcdstat` since you have `setDocExample` in the test? This fills in this section

**Comment by rtoma** [comment]
thanks for the support!

**Comment by shortdudey123** [comment]
np :)

**Pull request merged**